### DISCUSSION TITLE
When will the stable version of saleor 3.0 be released?

### DISCUSSION THREAD

**Discussion by Batgerel53** [saleor/saleor#10201] [Q&A] [answered]
I am currently using version saleor-3.0.0-b.33. Which version to use will be stable in the long term with less errors?
@karolkielecki , @maarcingebala , @orzechdev

**Comment by karolkielecki** [comment, marked as answer]
Hi @Batgerel53, 

We've recently released Saleor [3.5.1](https://github.com/saleor/saleor/releases/tag/3.5.1). That's the latest stable version. I encourage you to read all the changelogs since Saleor 3.0 to learn more about the new features and what has been changed and improved. https://github.com/saleor/saleor/releases

**Comment by Batgerel53** [comment]
Okay , Thank you

**Reply by Batgerel53** [reply]
Hi @karolkielecki ,
Sorry for asking again. 
If i understand correct, every upgraded version with "latest" could be stable right?

**Reply by karolkielecki** [reply]
Hi @Batgerel53, not always. Sometimes we release minor updates for older Saleor versions--just like we did recently. And they can be marked as `latest`. 
The newest version of Saleor is this one: https://github.com/saleor/saleor/releases/tag/3.6.3. Saleor 3.6+ introduced new features, including dynamic taxes, subscription support for synchronous webhook events, and more. We're currently working on v3.7, which should be released this month.
